app/javascript/helpers/custom-click-handler.js

```diff
@@ -0,0 +1,26 @@
+/** Function to handle the click events by redirecting the actions to js functions. */
+const customJsAction = ({ action }) => {
+  if (action.name === 'miqTreeActivateNode') {
+    const { nodeTree, nodeKey } = action;
+    if (nodeTree && nodeKey) {
+      window.miqTreeActivateNode(nodeTree, nodeKey);
+    }
+  } else if (action.name === 'miqOnClickSelectRbacTreeNode') {
+    if (action.treeId) {
+      window.miqOnClickSelectRbacTreeNode(action.treeId);
+    }
+  }
```

Comment:
Should there be another else case here for other things that might slip through?  Maybe console log an error or something?